app/src/main/java/com/spendster/presentation/authentication/AuthView.java

```diff
@@ -1,7 +1,7 @@
 package com.spendster.presentation.authentication;
 
-public interface AuthView {
-    void showNextActivity();
+import com.spendster.presentation.ErrorsView;
 
-    void showError(String message);
+public interface AuthView extends ErrorsView {
+    void showNextActivity();
```

Comment:
@SimonMaha It's better to name such methods not being coupled with "Activity" or "Fragment". "Screen" would be a better naming. The interface should not define whether the next screen is Activity or Fragment.